Dockerfile

```diff
@@ -44,6 +44,12 @@ COPY --from=builder /rules-content/content/ /rules-content
 # copy tutorial/fake rule to external rules to be hit by all reports
 COPY rules/tutorial/content/ /rules-content/external/rules
 
+RUN curl -L -o /usr/bin/haberdasher \
```

Comment:
Let's not use `latest` release. Please specify the exact version/

Reply:
Sure thing. I'll set it to v0.1.3, which is the latest right now.